### ISSUE TITLE
May the compilation methods of other platforms be added?

### ISSUE THREAD

**Issue by dalerkd** [tser-project/tser#3]
Linux or Windows? I think it will make it easier for more people to join.

**Comment by lanistor** [comment]
Yeah, this won’t cost too much time to support other platforms, just edit the build command.

**Comment by Dev-Sona** [comment]
Agreed. Not too sure what's stopping the OP. Also been 2 months since an update, and still obviously much to do. I might fork and fix some of these issues myself.
